packages/fxa-settings/src/components/ConnectedServices/MOCK_SERVICES.js

```diff
@@ -226,3 +178,62 @@ export const MOCK_SERVICES = [
     approximateLastAccessTimeFormatted: null,
   },
 ];
+
+const MOBILE_SYNC_SERVICE_MOCKS = [
+  {
+    clientId: 'e7ce535d93522896',
+    deviceId: 'f84f1ae16db91422f8c808ce3c5f3a04',
+    sessionTokenId: null,
+    refreshTokenId:
+      'b72fbbe94cd79b26c4ec0a8c3de237ca0c6beeea318874d041ade2c6c482c3a5',
+    isCurrentSession: false,
+    deviceType: 'mobile',
+    name: 'Firefox Lockwise',
+    createdTime: 1570221396000,
+    lastAccessTime: 1570221396619,
+    scope: ['https://identity.mozilla.com/apps/oldsync', 'profile'],
+    location: {
+      city: null,
+      country: null,
+      state: null,
+      stateCode: null,
+    },
+    userAgent: '',
+    os: null,
+    createdTimeFormatted: 'a month ago',
+    lastAccessTimeFormatted: 'a month ago',
+    approximateLastAccessTime: null,
+    approximateLastAccessTimeFormatted: null,
+  },
+  {
+    clientId: '3c49430b43dfba77',
+    deviceId: '9b16949ac8d5019ea08fdc0b82731a97',
+    sessionTokenId: null,
+    refreshTokenId:
+      '081ada3fb68fc41c97a60358bf6443c13c55a2ea5a85b2cac0c4c2970e2d22f7',
+    isCurrentSession: false,
+    deviceType: 'mobile',
+    name: 'A-C Logins Sync Sample',
+    createdTime: 1570038359000,
+    lastAccessTime: 1570038359594,
+    scope: ['https://identity.mozilla.com/apps/oldsync', 'profile'],
+    location: {
+      city: null,
+      country: null,
+      state: null,
+      stateCode: null,
+    },
+    userAgent: '',
+    os: null,
+    createdTimeFormatted: 'a month ago',
+    lastAccessTimeFormatted: 'a month ago',
+    approximateLastAccessTime: null,
+    approximateLastAccessTimeFormatted: null,
+  },
+];
+
+export const MOCK_SERVICES = [].concat(
+  DESKTOP_SYNC_MOCKS,
+  OAUTH_SERVICE_MOCKS,
+  MOBILE_SYNC_SERVICE_MOCKS
+);
```

Comment:
hmm don't think I understand this `concat` hipness.  Couldn't a plain [array literal] also work?

Reply:
hmm, `concat` is ancient and, now that you mention it, spread syntax is more modern and should do the same thing: `[...mocks, ...otherMocks]`

Reply:
ah, oops, sorry, those are arrays already.  Gotcha.